eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -66,11 +66,14 @@ import scala.concurrent.duration._
  * @param datadir          directory where eclair-core will write/read its data.
  * @param overrideDefaults use this parameter to programmatically override the node configuration .
  * @param seed_opt         optional seed, if set eclair will use it instead of generating one and won't create a seed.dat file.
+ * @param db               optional databases to use, if not set eclair will create the necessary databases
+ * @param disableBackup    disable automatic database backup (to speed up tests)
  */
 class Setup(datadir: File,
             overrideDefaults: Config = ConfigFactory.empty(),
             seed_opt: Option[ByteVector] = None,
-            db: Option[Databases] = None)(implicit system: ActorSystem) extends Logging {
+            db: Option[Databases] = None,
+            disableBackup: Boolean = false)(implicit system: ActorSystem) extends Logging {
```

Comment:
Why not make that a configuration parameter?

Reply:
That was my first idea, but in fact we really don't want users shooting themselves in the foot by disabling backup...that's why I only put it in the code itself, but if you think it's better in the conf I can do that.

Reply:
I think it's better for consistency, with a "*do not change this unless you know what you are doing*" comment. There are other parameters that can be used as footgun